processing_options.go

```diff
@@ -751,15 +763,30 @@ func parsePath(ctx context.Context, r *http.Request) (context.Context, error) {
 			return ctx, err
 		}
 	}
+	headers := &processingHeaders{
+		Accept:        "",
+		Width:         "",
+		ViewportWidth: "",
+	}
```

Comment:
Providing blank strings is redundant since strings are blank by default. This can be just `headers := &processingHeaders{}`